tests/all/memory.rs

```diff
@@ -0,0 +1,281 @@
+use anyhow::Result;
+use rayon::prelude::*;
+use wasmtime::*;
+
+fn module(engine: &Engine) -> Result<Module> {
+    let mut wat = format!("(module\n");
+    wat.push_str("(import \"\" \"\" (memory 0))\n");
+    for i in 0..=33 {
+        let offset = if i == 0 {
+            0
+        } else if i == 33 {
+            !0
+        } else {
+            1u32 << (i - 1)
+        };
+
+        for (width, instr) in [
+            (1, &["i32.load8_s"][..]),
+            (2, &["i32.load16_s"]),
+            (4, &["i32.load" /*, "f32.load"*/]),
+            (8, &["i64.load" /*, "f64.load"*/]),
+            (16, &["v128.load"]),
```

Comment:
This unconditionally adds a SIMD operation, which we don't support on s390x yet, causing the test to fail.

Can we make this part architecture-dependent for now?

Reply:
Oh sure yeah, it's also fine to just comment this out since it's not really the most critical part of the test anyway